docs/flashbots-auction/searchers/advanced/troubleshooting.mdx

```diff
@@ -201,18 +208,24 @@ Output:
 }
 ```
 
+:::note
+
+Parameters with `miner` in the name are retrofitted with Flashbots block builder data to maintain backwards compatibility. This nomenclature will be changed in a future release to accurately reflect PoS Ethereum architecture.
```

Comment:
You mention this in a few spots, but don't mention the new name: `fee_recipient`

Reply:
Looks good